lib/PVE/Storage/Custom/StorPoolPlugin.pm

```diff
@@ -1600,23 +1603,34 @@ sub activate_volume {
     my $vmid	    = $vol->{tags}->{VTAG_VM()};
     my $force_detach = 0;
     my $src_node    = _get_migration_source_node() || '';
+    my $parent_pid  = getppid;
```

Comment:
This is the wrong order. first set the death signal, THEN get the parent pid, otherwise you have a race condition of the parent dying between these two calls.

Reply:
Done, addressed in 66aed0d